ahnlich/ai/src/manager/mod.rs

```diff
@@ -167,27 +160,28 @@ impl Task for ModelThread {
 
 #[derive(Debug)]
 pub struct ModelManager {
-    models: HashMap<SupportedModels, mpsc::Sender<ModelThreadRequest>>,
+    models: HashMap<Model, mpsc::Sender<ModelThreadRequest>>,
```

Comment:
I think we may not need `model.clone()` if we leave this as `SupportedModels` or even `AIModel`. Since we already have the model itself within the `ModelThread` and what we receive via the external facing api `handle_request` is an `AIModel`, we don't need cloning, we can rather use the sparse copyable enums